src/ln/peer_handler.rs

```diff
@@ -520,6 +543,10 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 														self.initial_syncs_sent.fetch_add(1, Ordering::AcqRel);
 														local_features.set_initial_routing_sync();
 													}
+
+													if do_they_require_sync {
+														peer.last_synced_channel = 0; //set to larger than -1, means next sent message will trigger slow trickle of sync data
```

Comment:
You should insert into peers_needing_send as well so that it gets picked up by the next process_events.